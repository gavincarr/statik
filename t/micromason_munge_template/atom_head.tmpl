%# Atom head template - testing single level if and if-elsif
<?xml version="1.0" encoding="$blog_encoding"?>
<feed xmlns="http://www.w3.org/2005/Atom">
<title>$blog_title</title> 
<subtitle>$blog_description</subtitle> 
<link href="$url$path"/>
<updated>$index_updated_rfc3339</updated>
<author>
  <name>$author_name</name>
% if ($author_url) {
  <uri>$author_url</uri>
% }
% if (0) {
  <false>This shouldn't show up</false>
% } elsif ($author_email) {
  <email>$author_email</email>
% }
</author> 
<id>tag:$blog_id_domain,$blog_id_year:$path_abs</id>
<generator uri="https://github.com/gavincarr/statik" version="0.1">Statik</generator>

