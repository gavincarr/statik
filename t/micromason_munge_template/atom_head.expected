<?xml version="1.0" encoding="$blog_encoding"?>
<feed xmlns="http://www.w3.org/2005/Atom">
<title>$blog_title</title> 
<subtitle>$blog_description</subtitle> 
<link href="$url/$path"/>
<updated>$index_updated_iso8601</updated>
<author>
  <name>$author_name</name>
  <uri>$author_url</uri>
  <email>$author_email</email>
</author> 
<id>tag:$blog_id_domain,$blog_id_year:$path_abs</id>
<generator uri="https://github.com/gavincarr/statik" version="0.1">Statik</generator>
